### PR TITLE
fixes miniscope malus.

### LINKED ISSUE #247: mini railscope sets an accuracy malus when unscoping which it doesn't have before scoping
before scoping in accuracy_mod is 0; after scoping in it's +0.05; after unscoping it becomes -0.05

### PR DESCRIPTION
**Pull request by Ghommie**
:cl: Ghommie
bugfix: fixed the miniscope giving a negative accuracy bonus (ergo a malus) when unwielded.
/:cl:

Closes #247.

### CODE REVIEW

File: code/modules/projectiles/updated_projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -536,7 +536,7 @@ Defined in conflicts.dm of the #defines folder.
 /obj/item/attachable/scope/mini/activate_attachment(obj/item/weapon/gun/G, mob/living/carbon/user, turn_off)
 	if(turn_off)
 		if(G.zoom)
-			accuracy_mod = -config.low_hit_accuracy_mult
+			accuracy_mod = null
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/gun_attachables.dm]:
```suggestion
			accuracy_mod = 0
```

File: code/modules/projectiles/updated_projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -1111,7 +1111,7 @@ Defined in conflicts.dm of the #defines folder.
 		G.aim_slowdown += SLOWDOWN_ADS_SCOPE
 		G.wield_delay += WIELD_DELAY_FAST
 	G.update_slowdown()
-
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/gun_attachables.dm]: Avoid committing whitespace changes like this in future.

**Comment by Surrealaser** [comment]
I think this is actually addressed in the sniper laser targeter update which changes how scopes/zoom work

**Comment by Ghommie** [comment]
Oh, forgot to close this.... there, done, Surrealaser.

**Pull request closed without merging**